packages/ketchup/src/components/kup-echart/kup-echart.tsx

```diff
@@ -590,6 +593,59 @@ export class KupEchart {
         } as echarts.EChartsOption;
     }
 
+    #candleChart(){
+        
+        const y = this.#createY(),
+        date = this.#createX(),
+        arrayLength = date.length;
+        const answer = [], itemStyle = {}, colors = this.#setColors(arrayLength);
+
+        for (let i = 0; i < y['Open'].length; i++) {
+            answer.push([
+                parseInt(y['Open'][i]),
+                parseInt(y['High'][i]),
+                parseInt(y['Low'][i]),
+                parseInt(y['Close'][i])
+            ]);
+        }
+        let legend = {};
+        date.forEach((e,i)=>{
+          legend[e] = i;
+        })
+
+        colors.forEach((data, i)=>{
+            if(i==0){
+                itemStyle['color'] = data;
+                itemStyle['borderColor'] = data;
+            }
+            else{
+                itemStyle['color'+(i-1)]= data;
+                itemStyle['borderColor'+(i-1)] = data;
+
+            }
+        })
+
+        return  {
+            legend: this.#setLegend(legend),
+            color: this.#setColors(6),
```

Comment:
Colors should be only red and green, red if it's a loss and green if it's a gain.
For reference: https://upload.wikimedia.org/wikipedia/commons/thumb/e/ea/Candlestick_chart_scheme_03-en.svg/245px-Candlestick_chart_scheme_03-en.svg.png